gammapy/estimators/lightcurve.py

```diff
@@ -405,40 +401,31 @@ def run(self, datasets):
                 continue
 
             fp = self.estimate_time_bin_flux(datasets=datasets_to_fit)
-            fp_table = fp.to_table()
-
-            for column in fp_table.colnames:
-                if column == "counts":
-                    data = fp_table[column].quantity.sum(axis=1)
-                else:
-                    data = fp_table[column].quantity
-                row[column] = data
-
-            fp_table_flux = fp.to_table(sed_type="flux")
-            for column in fp_table_flux.colnames:
-                if "flux" in column:
-                    row[column] = fp_table_flux[column].quantity
-
-            rows.append(row)
+            # TODO: find a way to handle counts per dataset
```

Comment:
I did not get the issue... Is there an hidden approximation with this sum?

Reply:
Currently the counts are summed over all datasets, it would be good to keep the info per dataset for debugging. But this is future work and will not be addressed in this PR.